### ISSUE TITLE
Case sensitivity when parsing 'control' like files.

### ISSUE THREAD

**Issue by ebikt** [paultag/go-debian#57]
https://wiki.debian.org/RepositoryFormat#A.22Release.22_files:

> The "Release" files, "Packages" and "Sources" indicies, and files called "Index" that are used for translations and differences are control files, as defined in Policy, Chapter 5. In addition to the rules for control files, field names shall be generated using the case defined in this document, that is, code creating repositories shall be case-sensitive, but code reading repositories should not be case-sensitive.

I guess this aplies also to .changes files.

It will be neat to have option to either ignore case or require proper case (i.e., "raise error" when decoding key that matches key from struct only case-insensitively).

**Comment by paultag** [comment]
Good point! Perhaps enforce a canonization, since all of them follow the `Foo-Bar-Baz` convention; but maybe that isn't actually good enough.

I also wonder what the expected behavior for:

```
Foo: 1
foo: 1
``` 

is
